eZ/Publish/Core/Limitation/ParentOwnerLimitationType.php

```diff
@@ -128,20 +129,25 @@ public function evaluate(APILimitationValue $value, APIUserReference $currentUse
             return false;
         }
 
+        $hasMandatoryTarget = false;
         foreach ($targets as $target) {
+            if ($target instanceof Target\Version) {
```

Comment:
I intentionally put this code, because if we drop TargetAwareType, this code will be needed ;)

Reply:
Uh, please imagine someone adds yet another type of `Target` for completely different Limitation. It would require then to modify this code as well, which is a direct break on open-closed principle. 

This is completely opposite behavior than the one I had in mind for improved Limitation type handling. The Limitation should handle only what it knows about and for anything else either return ACCESS_ABSTAIN or ACCESS_GRANTED (only if not matching DENIED conditions implies it's safe to allow).

This particular case calls for checking negative conditions - refer to only what Limitation knows about, everything else handle in a common way.